lib/livebook_web/live/hub/new_live.ex

```diff
@@ -76,9 +76,22 @@ defmodule LivebookWeb.Hub.NewLive do
     <LayoutHelpers.layout current_page="/hub" current_user={@current_user} saved_hubs={@saved_hubs}>
       <div class="flex flex-col p-4 md:px-12 md:py-7 max-w-screen-md mx-auto space-y-8">
         <div>
-          <LayoutHelpers.title text="Add Hub" />
+          <LayoutHelpers.title text="Add Organization" />
```

Comment:
We generally use this casing

```suggestion
          <LayoutHelpers.title text="Add organization" />
```